guide/how-it-works/payment-request-formats.md

```diff
@@ -19,119 +19,235 @@ images:
 
 It could be tempting to refer to a Lightning invoice as a “lightning address.” Doing so could be confused with the [lightning address]({{'/guide/how-it-works/payment-request-formats/#lightning-addresses' | relative_url}}) protocol described below. Therefore, avoid calling an invoice an address. A “payment request” would be a better synonym for “invoice.”
 
 {% include tip/close.html %}
 
-### Offers ([BOLT 12](https://bolt12.org))
+### Invoice protocols
+
+Invoice protocols are static payment requests that dynamically generate and share [invoices]({{ '/guide/how-it-works/payment-request-formats/#invoice' | relative_url }}). When a sender scans or imports an invoice protocol request into their application, rather than the invoice being part of the imported data, the payer dynamically fetches a standard invoice from the user who created the invoice protocol request.
+
+The various types of invoice protocols offer flexibility and unique use cases not possible with single-use, standard invoices.
+
+#### Offers
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/offer.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/offer@2x.jpg"
+   alt-text = "A QR code of an offer"
+   width = 400
+   height = 500
+   layout = "float-right-desktop"
+%}
+
+Offers are an experimental invoice protocol developed by [Core Lightning](https://blockstream.com/lightning/). How they work on a technical level is defined in [BOLT 12](https://bolt12.org/). Offers are still a draft specification and are currently not widely supported.
+
+Offers can contain a wider range of meta data compared to standard invoices. This includes fiat currencies, requester names, payment limits (minimum and maximum), and recurrences (how often can a new request be generated). Some examples are outlined [here](https://bootstrap.bolt12.org/examples).
+
+Offer can also generate pull payments, say for refunds, and have the ability to create subscriptions. A subscription has the offer generate and share invoices with a subscriber as they need them.
 
-This draft specification has similarities to LNURL-Pay and LNURL-Withdraw, but uses the Lightning network itself as the communication channel.
+Unlike other invoice protocols, offers use native communication to share invoices. This means the invoices are shared within lightning and not out of band over private communication channels or through a web server (like [LNURL]({{ '/guide/how-it-works/payment-request-formats/#lnurl' | relative_url }})) making it more private, censorship resistant, and convenient.
 
-### [Lightning AMP invoices](https://docs.lightning.engineering/lightning-network-tools/lnd/amp) (Atomic Multi-Path)
+Offers do not need as much information as a standard invoice for a payment to be made. This has usability improvements such as smaller, easier to scan, QR codes.
```

Comment:
```suggestion
Offers do not need as much information as a standard invoice for a payment to be made. This has usability improvements, such as smaller, easier-to-scan QR codes.
```

@GBKS We're also going to need to remove that last comma and throw in some hyphens since we're using "easier to scan" as an adjective.